drivers/display/ssd1306.c

```diff
@@ -122,15 +112,9 @@ static int ssd1306_write_bus_spi(const struct device *dev, uint8_t *buf, size_t
 	int errno;
 
 	gpio_pin_set_dt(&config->data_cmd, command ? 0 : 1);
-	struct spi_buf tx_buf = {
-		.buf = buf,
-		.len = len
-	};
+	struct spi_buf tx_buf = {.buf = buf, .len = len};
 
-	struct spi_buf_set tx_bufs = {
-		.buffers = &tx_buf,
-		.count = 1
-	};
```

Comment:
Please stop unnecessary code reformatting.

Reply:
Restored it.
And also fix the follows.
https://github.com/zephyrproject-rtos/zephyr/compare/7b2136aabb677b5eec3e34fbaaacd305c2b29c89..48bb5d6f44bf05860258f87acc8b74676742d353#diff-7da6d958478082c2b7f2549d889797958fbf4226bbee0fadbf40e3907479afadL86-R82